auxin_cli/src/main.rs

```diff
@@ -89,8 +89,33 @@ pub fn launch_repl(_app: &mut crate::app::App) -> Result<()> {
 	panic!("Attempted to launch a REPL, but the 'repl' feature was not enabled at compile-time!")
 }
 
-#[tokio::main(flavor = "current_thread")]
```

Comment:
this is a simple macro but it's some hiddenness that I couldn't get behind

I read the implementation to figure out what it does